src/pages/slash-graphql/advanced-queries.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: "Advanced Queries with GraphQL+-"
+redirect_from:
+  - /slash-graphql/advanced-queries/
+---
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+*These are draft docs for Slash GraphQL, which is currently in beta*
+
+Slash GraphQL also supports running advanced queries with `GraphQL+-`, a query language that is unique to Dgraph. GraphQL+- should be used by advanced users who wish to make queries and mutations using existing Dgraph client libraries, either via the HTTP or gRPC endpoints. You can learn more about existing client libraries by following this [documentation](https://dgraph.io/docs/clients/).
+
+If you are getting started with Slash GraphQL, you might want to consider using our [GraphQL APIs](https://graphql.dgraph.io/doc/) instead. It will get you quickly started on the basics of using Slash GraphQL before you go into advanced topics. 
+
+Do note that Slash GraphQL does not allow you to change the schema or create new predicates via GraphQL+-. You will not be able to access the `/alter` endpoint. You can add your schema via Slash GraphQL (either UI or API) and then add your mutations according to your Schema using GraphQL+-.
```

Comment:
Please note. (either via the UI or via API [link to API])

Reply:
Please note that Slash GraphQL does not allow you to alter the schema or create new predicates via GraphQL+-. You will also not be able ta access the `/alter` endpoint or it's GRPC equivalent. Please add your schema through the GraphQL endpoint (either via the UI or via the Admin API), before accessing the data with GraphQL+-.